File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/ticket_item/exception/DuplicatedItemOptionGroupException.java
package band.gosrock.domain.domains.ticket_item.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class DuplicatedItemOptionGroupException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new DuplicatedItemOptionGroupException();

    private DuplicatedItemOptionGroupException() {
        super(TicketItemErrorCode.DUPLICATED_ITEM_OPTION_GROUP);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/ticket_item/exception/DuplicatedItemOptionGroupException.kt
package band.gosrock.domain.domains.ticket_item.exception

import band.gosrock.common.exception.DuDoongCodeException

class DuplicatedItemOptionGroupException private constructor() : DuDoongCodeException(TicketItemErrorCode.DUPLICATED_ITEM_OPTION_GROUP) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = DuplicatedItemOptionGroupException()
    }
}
